fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/codec/LuceneOptimizedCodec.java

```diff
@@ -55,12 +59,26 @@
 public class LuceneOptimizedCodec extends Codec {
 
     private final Lucene87Codec baseCodec;
-    private final LuceneOptimizedCompoundFormat compoundFormat;
-    private final LuceneOptimizedSegmentInfoFormat segmentInfoFormat;
-    private final LuceneOptimizedPostingsFormat postingsFormat;
-    private final LuceneOptimizedPointsFormat pointsFormat;
-    private final LuceneOptimizedDocValuesFormat docValuesFormat;
-    private final LuceneOptimizedStoredFieldsFormat storedFieldsFormat;
+    private final CompoundFormat compoundFormat;
+    private final SegmentInfoFormat segmentInfoFormat;
+    private final PointsFormat pointsFormat;
+    private final StoredFieldsFormat storedFieldsFormat;
+    private final PostingsFormat defaultPostingsFormat;
+    private final DocValuesFormat defaultDocValuesFormat;
+
+    private final PostingsFormat postingsFormat = new PerFieldPostingsFormat() {
```

Comment:
What is the purpose of adding the additional delegation on the `postingsFormat` and the `docValuesFormat`?

Reply:
It matches the current 87 Codec Implementation.